src/input.ts

```diff
@@ -123,7 +123,11 @@ function getSeparator(): string {
 }
 
 function getStrict(): boolean {
-    const strict = getRequiredInputString('strict');
+    const strict = getNonRequiredInputString('strict');

```

Comment:
>strict [](start = 10, length = 6)

what we get is string or bool?
"strict": false vs "strict": "false"

Reply:
Always string - The GitHub API `core.getInput` returns "false" in both cases

Reply:
Note:  `core.getInput` only returns strings

Looks like GitHub places the input into environment variables which get read back out by `getInput`

https://github.com/actions/toolkit/blob/master/packages/core/src/core.ts
`export function getInput(name: string, options?: InputOptions): string `